src/containers/MovieListContainer.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+
+import PropTypes from 'prop-types';
+
+import MovieList from '../components/MovieList';
+import {
+  request,
+  succsess,
+  failure,
+  reset,
+  setPagination,
```

Comment:
I don't think that name `setPaginaion` is good for this case.
To avoid name duplication you can use aliases when importing. For example `setPage as setPageAction`.